models/cam-fv/readme.rst

```diff
@@ -15,18 +19,19 @@ The DART system has supported data assimilation into CAM-FV continuously for man
 An interface to CAM-SE was added to DART in 2022.
 An interface to MPAS is being developed (contact us about the current status).
 
-.. |CAM6_Rean| replace:: 1 degree reanalysis wiki
-.. _CAM6_Rean: https://github.com/NCAR/DART/wiki/1-degree,-CAM6,-ensemble-reanalysis-for-CESM-experiments-(2011-thru-2019):-DATM,-hindcasts,-model-evaluation
+.. replaced by references.rst CAM6Rean_wiki
+   _CAM6_Rean: https://github.com/NCAR/DART/wiki/1-degree,-CAM6,-ensemble-reanalysis-for-CESM-experiments-(2011-thru-2019):-DATM,-hindcasts,-model-evaluation
 
 The flexibility of the DART environment has led to its use
 by graduate students, post-graduates, and scientists at universities and
 research labs to conduct data assimilation research. Others are using the
 products of data assimilation (analyses), which were produced at NSF NCAR
 using CESM+DART, to conduct related research. 
-The latest reanalysis is described in the DART |CAM6_Rean|_
+The latest reanalysis is described in the `DART 1 degree reanalysis wiki <CAM6rean_wiki_>`_
+
 
 The variety of research can be sampled on the DART  
-`Publications <https://dart.ucar.edu/pages/Publications.html>`__ page.
+`Publications <https://dart.ucar.edu/pages/Publications.html>`_ page.
```

Comment:
what's the goal of making this non-anonymous?

Reply:
In my limited reading I got the impression that anonymous is less robust, but now I see that it's an "embedded" URI and there are no other references to it (in this document), so anonymous is not any riskier than a named link or pointer.  I'll revert it.